src/Java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMTE_ElementalDuplicator.java

```diff
@@ -1,6 +1,9 @@
-/*
```

Comment:
I guess this is still WIP?

Reply:
Yeah, Just poked at it a bit earlier to kill time.